ddp-workspace/projects/ddp-pancan/src/variables.scss

```diff
@@ -0,0 +1,2 @@
+$header-height: 6rem;
+$footer-height: 14.94rem;
```

Comment:
#minor
Do we need such an accuracy there ?

Reply:
why not? any issues about that?

Reply:
Not an issue, just a strange value, as for me.

Reply:
I calculated it based on mockup sizes and keep 2 numbers after the dot. I hope it's more clear now.